fetch.bs

```diff
@@ -7480,6 +7485,48 @@ are:
  <li><p>Return <var>responseObject</var>.
 </ol>
 
+
+<p>The static
+<dfn method for=Response><code>json(<var>data</var>, <var>init</var>)</code></dfn> method steps
+are:
+
+<ol>
+ <li><p>If <var>init</var>["{{ResponseInit/status}}"] is not in the range 200 to 599, inclusive,
+ then <a>throw</a> a {{RangeError}}.
+
+ <li><p>If <var>init</var>["{{ResponseInit/status}}"] is a <a>null body status</a>, then
+ <a>throw</a> a {{TypeError}}.
+
+ <li><p>If <var>init</var>["{{ResponseInit/statusText}}"] does not match the
+ <a spec=http>reason-phrase</a> token production, then <a>throw</a> a {{TypeError}}.
+
+ <li><p>Let <var>responseObject</var> be the result of <a for=Response>creating</a> a {{Response}}
+ object, given a new <a for=/>response</a>, "<code>response</code>", and <a>this</a>'s
+ <a>relevant Realm</a>.
+
+ <li><p>Set <var>responseObject</var>'s <a for=Response>response</a>'s <a for=response>status</a> to
+ <var>init</var>["{{ResponseInit/status}}"].
+
+ <li><p>Set <var>responseObject</var>'s <a for=Response>response</a>'s
+ <a for=response>status message</a> to <var>init</var>["{{ResponseInit/statusText}}"].
+
+ <li><p>If <var>init</var>["{{ResponseInit/headers}}"] <a for=map>exists</a>, then
+ <a for=Headers>fill</a> <var>responseObject</var>'s <a for=Response>headers</a> with
+ <var>init</var>["{{ResponseInit/headers}}"].
+
+ <li><p>Set <var>responseObject</var>'s <a for=Response>response</a>'s <a for=response>body</a> to
+ the <a>UTF-8 encoding</a> of the result of running
+ <a>serialize a JavaScript value to a JSON string</a> on <var>data</var>.
```

Comment:
Infra has a ["serialize a JavaScript value to JSON bytes"](https://infra.spec.whatwg.org/#serialize-a-javascript-value-to-json-bytes) algorithm.